index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
   <body>
-    <nav>
-      <a class="logo" href="#">
+    <header class="header">
+      <a class="header--logo" href="#">
         <img src="images/keys.png" alt="keyboard" />
       </a>
-      <ul class="nav-links-cont">
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+      <nav class="header--nav">
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-links" href="#">
-          <li>Link</li>
+
+        <a class="header--link" href="#">
+          Link
         </a>
-        <a class="nav-call" href="#">
-          <button>Button</button>
+
+        <a class="button--small header--button">
```

Comment:
Instead of using two elements here, we can just use the class we were going to put on the button on the a tag. One less item in the markup and we don't need to do anything special to the styling either!

Reply:
This is how I used to do it, but I thought the idea is that if you have a button, you want to use the button tag for accessibility reasons. It's more markup though, of course, so if this way is better then I'm happy to skip using the button tag and just use an a tag

Reply:
Yeah I think it has more to do with using the right tags in terms of semantics. Usually when i think of a button, I think of something that can be clicked but doesn't take you to a new page. 

A good example would be a burger nav. It shouldn't be an <a> tag because those are for links that take you to a different page so it makes the most sense to use a button


In this case though, we're probably going to want to link to a different page so it makes sense to use an <a> tag

As far as accessibility, I don't think one is better than the other but using them semantically will help visually impaired people a ton!!!

This is because something like a Screen Reader will tell the user what kind of an element they are currently on and they have expectations on how things should behave. So they might get confused if they click a <button> tag and get taken to a new page